### PR TITLE
feat: Flag both policies and libraries for embedding.

### PR DESCRIPTION
**Pull request by simar7**
Reverts: #926

Signed-off-by: Simar <simar@linux.com>

### CODE REVIEW

**Review by owenrumney — DISMISSED** [review]

**Comment by simar7** [comment]
Can someone with write access merge this in if it all looks good? I don't seem to have it.

**Comment by simar7** [comment]
ping @liamg and @owenrumney

**Review by liamg — APPROVED** [review]

**Pull request merged**
